common/mqtt_client/examples/sawtooth_publisher.rs

```diff
@@ -178,8 +178,8 @@ async fn publish_multi_topic(
 
 async fn listen_command(mut messages: Box<dyn MqttMessageStream>) {
     while let Some(message) = messages.next().await {
-        debug!("C8Y command: {:?}", message.payload);
-        if let Some(cmd) = std::str::from_utf8(&message.payload).ok() {
+        debug!("C8Y command: {:?}", message.payload_utf8());
```

Comment:
maybe it would be good to rename `payload_utf8` to just `payload_str`. "utf8" is often used to refer to a byte slice or byte vector that contains utf8 characters.

Reply:
Agree. I expect that function to return a `str` (hence an utf8 content, but the reverse is not true).